feature/kanban_board/client/assets/stylesheets/lib/_flex.scss

```diff
@@ -0,0 +1,35 @@
+/*Flex properties*/
+.flex{
+    display: flex;
+}
+.flex-between {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+.flex-around{
+    display: flex;
+    justify-content: space-around;
+}
+.flex-start {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+}
+.flex-center {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.flex-end{
+    display: flex;
+    justify-content: flex-end;
+}
+.align-start {
+    display: flex;
+    align-items: flex-start;
+}
+.flex-column{
+    display : flex;
+    flex-direction: column;
+}
```

Comment:
Try to keep a blank extraline an after each line

Reply:
Resolved